frontend/src/util/apiWrapper.js

```diff
@@ -129,7 +130,14 @@ export const fetchFileSummary = (uuid, path) => {
     };
     const url =
         '/rest/bundles/' + uuid + '/contents/blob' + path + '?' + new URLSearchParams(params);
-    return get(url, { headers: { Accept: 'text/plain' } });
+    return get(
+        url,
+        {
+            headers: { Accept: 'text/plain' },
+        },
+        // need to define transformResponse to prevent json being parsed due to the issue discussed in: https://github.com/axios/axios/issues/811
```

Comment:
```suggestion
        // Get the raw text response and don't parse JSON. Need to define transformResponse to prevent JSON being parsed due to the issue discussed in: https://github.com/axios/axios/issues/811
```